src/taproot.rs

```diff
@@ -413,15 +287,15 @@ mod test {
         let mut sig_agg =
             v2::SignatureAggregator::new(Nk, T, A.clone()).expect("aggregator ctor failed");
         let aggregate_public_key = sig_agg.poly[0];
-        let tweaked_public_key = compute::tweaked_public_key(&aggregate_public_key, &merkle_root);
-
-        let (nonces, sig_shares) =
-            test_helpers::sign_tweaked(&msg, &mut S, &mut rng, &tweaked_public_key);
-        let sig = match sig_agg.sign_merkle_root(&msg, &nonces, &sig_shares, &key_ids, &merkle_root)
-        {
-            Err(e) => panic!("Aggregator sign failed: {:?}", e),
-            Ok(sig) => sig,
-        };
+        let tweaked_public_key =
```

Comment:
Might not be a bad idea to add a test with "none" for the Merkle Root.

Reply:
Done, for v1 and v2